Makefile

```diff
@@ -54,14 +54,14 @@ CMPR_FLAG = $(if $(findstring /Zc, $(dir $<)),c,)
 
 # Change many things if bit width isn't 64
 $(SRCDIR64)/%.elf: $(SRCDIR64)/%.$(SEXT) 
-	riscv64-unknown-elf-gcc -g -o $@ -march=rv64gq$(CMPR_FLAG)_zfa_zba_zbb_zbc_zbs_zfh_zicboz_zicbop_zicbom_zicond -mabi=lp64 -mcmodel=medany \
+	riscv64-unknown-elf-gcc -g -o $@ -march=rv64gq$(CMPR_FLAG)_zcb_zfa_zba_zbb_zbc_zbs_zfh_zicboz_zicbop_zicbom_zicond -mabi=lp64 -mcmodel=medany \
```

Comment:
Should `zcb` be part of `CMPR_FLAG` to ensure compressed instructions are only generated when appropriate?

Reply:
Yup, it makes sense. Thanks for your insight. Appreciated!
I made the required changes. Please have a look.